nodejs/week1/circle.js

```diff
@@ -0,0 +1,21 @@
+class circle {
```

Comment:
A naming convention for JavaScript classes is with a PascalCase in contrast to other JavaScript data structures.
So it should be Circle in your case.